plugins/protocolbuffers/pyi/v27.2/buf.plugin.yaml

```diff
@@ -0,0 +1,18 @@
+version: v1
+name: buf.build/protocolbuffers/pyi
+plugin_version: v27.2
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Python Stubs. Generates stub files for message and enum types.
+output_languages:
+  - python
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v27.2/LICENSE
+registry:
+  python:
+    package_type: "stub-only"
+    # https://github.com/protocolbuffers/protobuf/blob/v27.2/python/dist/setup.py#L78
+    requires_python: ">=3.8"
+    deps:
+      # https://pypi.org/project/protobuf/
+      - "protobuf~=5.27"
+      - "types-protobuf"
```

Comment:
```suggestion
      - "types-protobuf~=5.27"
```

Reply:
cc @stefanvanburen - We can now put a more specific types version.

Reply:
yeah they're a bit [irregular in their releases](https://pypi.org/project/types-protobuf/#history), but I see that they have the 5.27 version up now: https://pypi.org/project/types-protobuf/5.27.0.20240626/. (manual PRs: https://github.com/python/typeshed/pull/12197)